books/RayTracingInOneWeekend.html

```diff
@@ -3302,9 +3303,11 @@
 
 Total Internal Reflection
 --------------------------
-That definitely doesn't look right. One troublesome practical issue is that when the ray is in the
-material with the higher refractive index, there is no real solution to Snell’s law, and thus there
-is no refraction possible. If we refer back to Snell's law and the derivation of $\sin\theta'$:
+One troublesome practical issue with refraction is that there are ray angles for which no solution
+is possible using Snell's law. This happens when the ray goes into a medium of lower index of
```

Comment:
Slight rephrasing of the second sentence:

_When the ray enters a medium of lower index of refraction at a sufficiently glancing angle, it can refract with an angle greater than 90&deg;._

Reply:
Done.